src/ManageCourses.Domain/Migrations/20181212152132_AddMoreUcasFieldsToCourse.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace GovUk.Education.ManageCourses.Domain.Migrations
+{
+    public partial class AddMoreUcasFieldsToCourse : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "english",
+                table: "course",
+                nullable: true);
+
+            migrationBuilder.AddColumn<string>(
```

Comment:
Shouldn't this be a boolean? Then check for 'Y' or empty in the cell

Reply:
Good suggestion

Reply:
Done